packages/icon/README.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Icon
+description:
+---
+
+Make use of our Design System's icon library, anywhere in your app.
```

Comment:
I'm not sure this is really what we're going for here. This is a very specific set of icons which support a limited set of use cases, NOT a general use set of icons for whatever you need.

```suggestion
Make use of our Design System's icon library, anywhere in your app.
```